### PR TITLE
Add http-kafka asyncapi example

### PR DESCRIPTION
**Pull request by bmaidics**

### CODE REVIEW

**Comment by bmaidics** [comment]
Should only be merged once https://github.com/aklivity/zilla/pull/1072 is merged and released in Zilla.

File: http.kafka.asyncapi.proxy/README.md
Comment (jfallows) [inline http.kafka.asyncapi.proxy/README.md:1]: Rename scenario to `asyncapi.http.kafka.proxy` instead.

All `asyncapi` examples should follow this naming convention.

File: http.kafka.asyncapi.proxy/README.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# http.kafka.asyncapi.proxy
```
Comment (jfallows) [inline http.kafka.asyncapi.proxy/README.md:1]: This will need to adjust to the new name.

File: http.kafka.asyncapi.proxy/README.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+## Running locally
+
+This example can be run using Docker compose or Kubernetes. The setup scripts are in the [compose](./docker/compose) and [helm](./k8s/helm) folders respectively and work the same way.
+
+You will need a running kafka broker. To start one locally you will find instructions in the [kafka.broker](../kafka.broker) folder. Alternatively you can use the [redpanda.broker](../redpanda.broker) folder.
+
+### Setup
+
+Wether you chose [compose](./docker/compose) or [helm](./k8s/helm), the `setup.sh` script will:
```
Comment (jfallows) [inline http.kafka.asyncapi.proxy/README.md:14]:
```suggestion
Whether you chose [compose](./docker/compose) or [helm](./k8s/helm), the `setup.sh` script will:
```

File: http.kafka.asyncapi.proxy/docker/compose/teardown.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+NAMESPACE="${NAMESPACE:-zilla-http-kafka-asyncapi-proxy}"
```
Comment (jfallows) [inline http.kafka.asyncapi.proxy/docker/compose/teardown.sh:4]: Please check all scripts etc for rename of example scenario.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: asyncapi.http.kafka.proxy/docker/compose/docker-compose.yaml [hunk 4]
```diff
@@ -1,16 +1,12 @@
 version: '3'
 services:
   zilla:
-    image: ghcr.io/aklivity/zilla:${ZILLA_VERSION}
+    image: ghcr.io/aklivity/zilla:develop-SNAPSHOT
     container_name: zilla
-    pull_policy: always
```
Comment (vordimous) [inline asyncapi.http.kafka.proxy/docker/compose/docker-compose.yaml]: pull_policy is needed when the build isn't a local build. probably best to just comment it out in the future.

**Review by jfallows — APPROVED** [review]

**Review by vordimous — APPROVED** [review]

File: asyncapi.sse.proxy/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:22.04
+
+ENTRYPOINT ["/usr/app/node_modules/.bin/sse-server"]
```
Comment (vordimous) [inline asyncapi.sse.proxy/Dockerfile]: You can delete this file. I added an sse-server image: https://github.com/aklivity/zilla-demos/pkgs/container/extras-sse-server

File: asyncapi.sse.proxy/chart/templates/service-sse-server.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      app.kubernetes.io/instance: sse-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: sse-server
+    spec:
+      containers:
+        - name: sse-server
+          args: ["-v", "-p", "8001", "-i", "7001"]
+          image: "zilla-examples/sse-server:latest"
```
Comment (vordimous) [inline asyncapi.sse.proxy/chart/templates/service-sse-server.yaml]:
```suggestion
          image: "ghcr.io/aklivity/extras-sse-server:sha-42ad67e"
```

Comment (vordimous) [reply asyncapi.sse.proxy/chart/templates/service-sse-server.yaml]: This is the same code just with the image published to your package registry

File: asyncapi.sse.proxy/setup.sh [hunk 7]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex
+
+# Verify SSE Server image already available locally
+docker image inspect zilla-examples/sse-server:latest --format 'Image Found {{.RepoTags}}'
```
Comment (vordimous) [inline asyncapi.sse.proxy/setup.sh]: you can remove this.

File: asyncapi.sse.proxy/README.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+Listens on http port `7114` and will stream back whatever is published to `sse_server` on tcp port `7001`.
+
+### Requirements
+
+- bash, jq, nc
+- Kubernetes (e.g. Docker Desktop with Kubernetes enabled)
+- kubectl
+- helm 3.0+
+
+### Build the sse-server docker image
```
Comment (vordimous) [inline asyncapi.sse.proxy/README.md]: don't need to do this step anymore

**Review by vordimous — CHANGES_REQUESTED** [review]

File: asyncapi.sse.proxy/setup.sh (hunk 7 above)
Comment (vordimous) [inline asyncapi.sse.proxy/setup.sh]:
```suggestion
```

File: asyncapi.sse.proxy/README.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+output:
+
+```text
+...
+ => exporting to image                                                                                                                                                                       1.4s 
+ => => exporting layers                                                                                                                                                                      1.4s 
+ => => writing image sha256:104abb7e5a4389c50d02aaf5d9bb2fef883c82e066ac2b400c9039b35086efcc                                                                                                 0.0s 
+ => => naming to docker.io/zilla-examples/sse-server:latest
+```
+
```
Comment (vordimous) [inline asyncapi.sse.proxy/README.md]:
```suggestion
```

**Pull request merged**